source/gui/addonStoreGui/viewModels/addonList.py

```diff
@@ -93,9 +93,15 @@ class AddonListField(_AddonListFieldData, Enum):
 	)
 	publicationDate = (
 		# Translators: The name of the column that contains the publication date of the add-on.
-		pgettext("addonStore", "Publication Date"),
+		pgettext("addonStore", "Publication date"),
 		50,
 	)
+	installDate = (
+		# Translators: The name of the column that contains the installation date of the add-on.
+		pgettext("addonStore", "Install date"),
+		50,
+		frozenset({_StatusFilterKey.AVAILABLE}),
```

Comment:
Please, don't try to show the installation date in the updatable add-ons tab. Check that, if you switch to this tab, now an error is produced. Also, update the description of this PR accordingly.

```suggestion
		frozenset({_StatusFilterKey.AVAILABLE, _StatusFilterKey.UPDATE}),
```